backend/src/openarchiefbeheer/zaken/utils.py

```diff
@@ -177,14 +178,14 @@ def format_selectielijstklasse_choice(resultaat: Resultaat) -> DropDownChoice:
 
 
 @lru_cache
-def retrieve_selectielijstklasse_choices(process_type_url: str) -> list:
+def retrieve_selectielijstklasse_choices(query_params: HashableDict) -> list:
```

Comment:
Since the `get` supports params being None, I think it makes sense to add it to the types. 
```suggestion
def retrieve_selectielijstklasse_choices(query_params: HashableDict | None) -> list:
```